src/index.html

```diff
@@ -6,9 +6,39 @@
         <title>Events Manager</title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" 
+        integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" 
+        crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     </head>
     <body>
         <div id="App">
         </div>
+
+        <script>
+            // Your web app's Firebase configuration
+            var firebaseConfig = {
```

Comment:
@magicmarie these configurations should not be in this file. They should be in a .js file in your src directory. And also use env variables for the config values.

Reply:
oh yes.. intended to do that..